docs/faq.md

```diff
@@ -15,6 +19,31 @@ using standard HTTP compression negotiation and eliminating binary framing from
 the body, the Connect protocol lets us make unary RPCs with web browsers, cURL,
 or any other HTTP client.
 
+### How do I serve gRPC services when they are already written in a language that doesn't have a Connect implementation?
```

Comment:
I think the title here is a little unclear. Is the intention behind the question better captured by something like, "How can I use the Connect protocol to call existing gRPC servers?"

I think that the answer should re-emphasize that most Connect implementations support the gRPC protocol, so you have a choice:

1. Use the Connect runtime, but configure your client to use the gRPC _protocol_.
2. Use the Connect runtime and Connect protocol, and have Envoy automatically handle the Connect-to-gRPC translation.